types/lightship/index.d.ts

```diff
@@ -10,30 +10,57 @@ import { Server } from 'http';
  * A teardown function called when shutdown is initialized.
  * @param userConfiguration User configuration object
  */
-export function createLightship(userConfiguration?: UserConfigurationType): LightshipType;
+export function createLightship(configuration?: ConfigurationInputType): LightshipType;
 
 export type BeaconContextType = object;
 
 export interface BeaconControllerType {
     die: () => Promise<void>;
 }
 
-export interface UserConfigurationType {
-    /* Run Lightship in local mode when Kubernetes is not detected. Default: true. */
+export interface ConfigurationInputType {
+    /**
```

Comment:
Change the comments to be in JSDoc/TSDoc format (i.e., `/**`, not `/*`), so that editors display them